src/redisraft.h

```diff
@@ -418,9 +418,10 @@ typedef struct RedisRaftCtx {
     unsigned long snapshotreq_received;          /* Number of received snapshotreq messages */
     unsigned long exec_throttled;                /* Number of command executions throttled due to slow execution */
 
-    int entered_eval;             /* handling a lua script */
-    RedisModuleDict *locked_keys; /* keys thar have been locked for migration */
-    RedisModuleDict *acl_dict;    /* maps acl strings to RedisModuleUser * objects */
+    int entered_eval;                     /* handling a lua script */
+    RedisModuleDict *locked_keys;         /* keys thar have been locked for migration */
```

Comment:
```suggestion
    RedisModuleDict *locked_keys;         /* keys that have been locked for migration */
```
let's fix this on the way